### PR TITLE
Table Block variants added

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes 

Test URLs:
- Original: https://www.sunstar.com/jp/newsroom/news/20230720/
- Before: https://main--sunstar--hlxsites.hlx.live/_drafts/piyush/20230301-2
- After: https://text-block-variant--sunstar--hlxsites.hlx.live/_drafts/piyush/20230301-2

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation
------------- | -------------
 For e.g. cards | [doc-link](https://main--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=0)

- [ ] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
 For e.g. cards (grid)  | [doc-link](https://main--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=2)

### CODE REVIEW

File: blocks/table/table.css [hunk 1]
```diff
@@ -24,6 +24,10 @@ main .table .table-item div {
     background-color: var(--white);
 }
 
+main .table.left-align .table-item div {
```
Comment (sdmcraft) [inline blocks/table/table.css:27]: We can add a `.block.left-align` style. I think it can be useful across blocks.

**Review by sdmcraft — COMMENTED** [review]
How about you create a "table" in Word and put content in it? If it looks similar, it would be more intuitive for authors.

**Comment by jindaliiita** [comment]
> How about you create a "table" in Word and put content in it? If it looks similar, it would be more intuitive for authors.

Didn't exactly get this point as table in word will represent the block

**Comment by sdmcraft** [comment]
> Didn't exactly get this point as table in word will represent the block

Ah yes you are right. I had used it inside a block where it works like a table but when used directly, its treated as a block. How about having a boxed variant of text block with borders

File: styles/styles.css [hunk 2]
```diff
@@ -498,6 +498,12 @@ main .block {
   margin-right: auto;
 }
 
+main .block.border {
+  border: 1px solid;
+  padding: 0.5rem;
+  margin-bottom: 1rem;
```
Comment (sdmcraft) [inline styles/styles.css:504]: Probably these are applicable only when there's a `text` box with border, so should move to `text` block. The `border` style should ideally just impose a border.

Comment (jindaliiita) [reply styles/styles.css:504]: Ok got it

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**